proxy/stat/server.py

```diff
@@ -6,7 +6,33 @@
 from common.stat.metric import StatRegistry, StatSummary, StatGauge
 from common.stat.prometheus import PrometheusServer
 from common.utils.process_pool import ProcessPool
-from .api import STATISTIC_ENDPOINT, TxPoolData, TxFailData, TxDoneData
+from .api import (
+    OpResourceNeonBalanceData,
+    OpResourceSolBalanceData,
+    STATISTIC_ENDPOINT,
+    TxPoolData,
+    TxFailData,
+    TxDoneData,
+)
+
+
+class OpResourceStatApi(AppDataApi):
+    name: ClassVar[str] = "ProxyStatistic::OpResource"
+
+    def __init__(self, registry: StatRegistry):
+        super().__init__()
+        self._op_res_neon_balance = StatGauge("op_res_neon_balance", "Operator resource NEON balance", registry=registry)
+        self._op_res_sol_balance = StatGauge("op_res_sol_balance", "Operator resource SOL balance", registry=registry)
```

Comment:
The same is here.
The operator can earn not only NEON, he can also receive SOL tokens as a payment.

So, it will be better to give another name:
- "op_res_spending_tokens_balance"

Reply:
Again with the client, I have called it `op_resource_earned_tokens_balance`.